src/mapping/query/function/format_timestamp.rs

```diff
@@ -18,20 +18,21 @@ impl FormatTimestampFn {
 }
 
 impl Function for FormatTimestampFn {
-    fn execute(&self, ctx: &Event) -> Result<Value> {
-        let format = match self.format.execute(ctx)? {
+    fn execute(&self, ctx: &Event) -> Result<QueryValue> {
+        let format = match self.format.execute(ctx)?.into() {
             Value::Bytes(b) => String::from_utf8_lossy(&b).into_owned(),
             v => unexpected_type!(v),
         };
 
         self.query
             .execute(ctx)
-            .map(|v| match v {
+            .map(|v| match v.into() {
                 Value::Timestamp(ts) => ts,
-                _ => unexpected_type!(v),
+                v => unexpected_type!(v),
             })
             .and_then(|ts| try_format(&ts, &format))
             .map(Value::from)
+            .map(Into::into)
```

Comment:
Can we add an `impl<T: Into<Value>> From T for QueryValue`? That way we can directly go from any `T: Into<Value>` to `QueryValue`.

If not, perhaps we should just implement a custom `QueryValue::from_value`, which would do the same.

Reply:
The `impl<T: Into<Value.>` keeps giving *conflicting implementations* errors unfortunately, but a `QueryValue::from_value` seems to work nicely.